src/commands/build/internal/getLaunchPageCode.ts

```diff
@@ -0,0 +1,49 @@
+import { ServerType } from '@sasjs/utils/types'
+
+const SAS9Code = (streamServiceName: string) => `
+options notes;
+data _null_;
+ format url $256.; 
+ rc=METADATA_GETURI("Stored Process Web App",url);
+ url=coalescec(url,"localhost/SASStoredProcess");
+ putlog "NOTE: SASjs Streaming App Created!!  Check it out here:" ;
```

Comment:
@allanbowe Is this text okay? i.e. do we _really_ need two exclamation marks?

Reply:
that's my input!  I have been told in the past that I'm a heavy user of exclamation marks.   Will reduce